imagersite/imager_profile/templates/profile.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+{% block content %}
+
+<div class='imager-profile'>
+<p id='join_date'>Joined on {{ user.date_joined|date:"D d M Y" }}</p>
+<h2>{{ user.first_name }} {{ user.last_name }}</h2>
+<p><a href="{{ user.profile.website }}"> {{user.profile.website }}</a></p>
+
+
+<p id='usage'>
+You have <strong>{{ user.profile.albums.all|length }}</strong> albums<br/>
+you have a total of <strong>{{ user.profile.photos.all|length }}</strong> photos.
```

Comment:
We discussed in class why it might be a better idea to relate the album and photos to a User rather than a Profile.